about.html

```diff
@@ -0,0 +1,121 @@
+                        <li id="campaign"><a class="menu__item" href="#">Campaign</a></li>
+                    </ul>
+
+                </div>
+                
+            </div>
+
+        </nav>
+        <main>
+            <section >
+                <div class="hero">
+                    <div class="hero-header-about">
+                        <h1 id="hero-top-about"><span>"Life is better at full throttle!"</span><br>WRC CAR-NNECT SUMMIT 2023</h1>
+                        <p id="hero-paragraph-about">The Global Summit brings together the community of experts,rally racers and common affiliate
+                            networks from all over the world every two years.This year we hope to expand our invitation list including organisations
+                            and individuals.</p>
+                            <div class="contact">
+                            <p>Please contact us per Email for further questions about WRC Car-nnect 2023! <span>carnnect2023@gmail.org</span></p>
+                            </div>
+                    </div>
+                </div>
+            </section>
+    
+            <section class="sec-logo">
+                <div class="logo_top">
+                    <h3>WRC Car-nect Global Summit 2023 Logo</h3>
+                </div>
+                
+                <div class="indicator"><span></span></div>
+                <p>The logo of WRC car-nnect was decided through the logo competition from 8,January to 7,July.</p>
+                <div class="logo-container">
+                    <img src="images/logo.png" alt="logo" id="big-logo">
+                </div>
+            </section>
+    
+            <section class="past-summits">
+                <h3>See the past WRC Car-nnect Summits</h3>
+                <div class="indicator"><span></span></div>
+                <p>Take a look at the past two WRC Car-nnect Summit which took place in Nairobi and in Mombasa</p>
+                <div class="pasts">
+                    <div class="psts1">
+                        <h4 id="psts-h">2019</h4>
+                        <p id="psts-p">Masinga TT National Summit,Mombasa</p>
+                    </div>
+                    <div class="psts2">
+                        <h4 id="psts-h">2021</h4>
+                        <p id="psts-p">Carchella National Summit,Machakos</p>
+                    </div>
+                    
+                </div>
+    
+            </section>
+    
+    
+    
+    
+            <section class="partners">
+                <div class="partners_top">
+                    <h3>Partners</h3>
+                </div>
+                <div class="indicator"><span></span></div> 
+                <div class="partners-logo">
+                    <img src="images/ford.svg" alt="">
+                    <img src="images/hyundai.svg" alt="">
+                    <img src="images/toyota.svg" alt="">
+                </div> 
+    
+    
+            </section>
```

Comment:
- [ ] The partner section is not included on the desktop version of about page. It is only included on the mobile version. Kindly hide it on the desktop version. 
- [ ] Also the footer background-color is dark not white. Kindly fix that too.


### Your Design
![your-about-footer](https://user-images.githubusercontent.com/26597432/226090762-f31c8c15-08df-49ff-94a5-1c7e7ad2840d.png)

### Template Design
![temp-about-footer](https://user-images.githubusercontent.com/26597432/226090742-42597bd4-ba99-4b4d-a61d-95491a6c17f9.png)